pkg/api/types.go

```diff
@@ -513,8 +523,8 @@ type PodSpec struct {
 // PodStatus represents information about the status of a pod. Status may trail the actual
 // state of a system.
 type PodStatus struct {
-	Phase PodPhase `json:"phase,omitempty"`
-
+	Phase        PodPhase        `json:"phase,omitempty"`
+	ServicePhase PodServicePhase `json:"servicePhase,omitempty"`
```

Comment:
I would like this to look similar to NodeConditions:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L766

"Phase" is reserved for monotonic progressions. "Conditions" may change back and forth. Also, I expect to add more conditions in the future. And, I'd like to be able to add more information in the future, such as transition time, reason, message, etc.